config/sync/honeypot.settings.yml

```diff
@@ -1,25 +1,12 @@
 protect_all_forms: false
-log: false
+log: true
 element_name: email
-time_limit: 5
-expire: 300
+time_limit: 0
 form_settings:
   user_register_form: false
   user_pass: false
   contact_message_feedback_form: true
   contact_message_page_feedback_form: true
   contact_message_personal_form: false
-  node_article_form: false
```

Comment:
Again. Curious why this was touched

Reply:
It was added when bouncing between branches. Needs to be tossed out.